test/runtime/Policies.go

```diff
@@ -839,7 +799,15 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 					"%s"
 				]
 			}]
-		}]`, httpd1Label, httpd2Label, ipv4Prefix, ipv6Prefix)
+		},
+		{
+			"endpointSelector": {"matchLabels":{"%s":""}},
+			"egress": [{
+				"toEndpoints":  [
+					{"matchLabels":{"%s":""}}
+				]
+			}]
+		}]`, httpd1Label, httpd2Label, ipv4Prefix, ipv6Prefix, "id.app3", httpd1Label)
```

Comment:
- Could you use "%[1]s" instead repeating httpd1Label
- Could you make "id.app3" a const to avoid hardcoding.